docs/source/man.rst

```diff
@@ -141,10 +141,13 @@ To create a credential, supply SSH credentials as either a username-password pai
 
   Prompts for the password to be used when running a command that uses sudo on the systems to be scanned.
 
+``--become_method``
```

Comment:
this shouldn't be a prompt you should be able to pass in `sudo` or `su`, etc. at the command line